sdk/eventhub/Azure.Messaging.EventHubs/tests/Primitives/EventProcessorTests.PartitionProcessing.cs

```diff
@@ -1664,7 +1664,8 @@ public async Task CreatePartitionProcessorProcessingTaskLogsTheStartingPosition(
                     mockProcessor.Object.Identifier,
                     mockProcessor.Object.EventHubName,
                     mockProcessor.Object.ConsumerGroup,
-                    expectedEevntPosition.ToString()),
+                    expectedEevntPosition.ToString(),
```

Comment:
nit (def in L1631)
```suggestion
                    expectedEventPosition.ToString(),
```

Reply:
Nice catch!   Applied by hand to get the definition and other uses.